tests/common.go

```diff
@@ -699,6 +699,10 @@ func ValidateContext(ctx *scheduler.Context, errChan ...*chan error) {
 		}
 
 		Step(fmt.Sprintf("validate %s app's volumes", ctx.App.Key), func() {
+			// In case of tektoncd skip the volume validation
```

Comment:
Add the reason for skipping volume validation in the doc string

Reply:
Done